articles/Setup-Windows-Python.md

```diff
@@ -38,7 +38,12 @@ The default CNTK math library is the [Intel Math Kernel Library (Intel MKL)](htt
 
 ### OPTIONAL. OpenCV
 
-CNTK 2.2 requires [Open Source Computer Vision (OpenCV)](http://opencv.org/) to be installed but it is optional for CNTK 2.3+. Follow [this page](./Setup-OpenCV-on-Windows.md) to install it.
+CNTK 2.2 requires [Open Source Computer Vision (OpenCV)](http://opencv.org/) to be installed, but it is optional for CNTK 2.3+. Follow [this page](./Setup-OpenCV-on-Windows.md) to install it.
```

Comment:
Great info, I was wondering what are the components that require OpenCV :)

Do you happen to know what components require the MKL library ? That info might also help users know when they should install MKL before using CNTK

Maybe this is not required if we consider MKL as an required component. However, I've seen users saying that "I am not using MKL features, so I don't need to install it"